### ISSUE TITLE
[request] Coq selective/filtered import should support blacklists

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#15017]
I'd like an equivalent of Agda's `hiding` when doing [filtered import](https://coq.inria.fr/refman/language/core/modules.html#grammar-token-filtered_import), something like `Import foo(-(id, compose))`, which has the effect of adding everything not in the inner parentheses.

**Comment by Blaisorblade** [comment]
Yes @Alizter — `A.Foo` is already visible and `Import A(-Foo).` makes only `Bar` visible. OTOH that is not equivalent to local definitions, since a later `Import A(Foo)` or `Import A(-Bar).` would make `Foo` visible as well.

More formally, the OP seems to imply a full specification (but I'm probably missing some corner cases):

> `Import foo(-(id, compose))`, which has the effect of adding everything not in the inner parentheses.

IOW, if `foo` contains a set `X` of (non-local) members, `Import foo(-Y)` should import identifiers in the the set `Z = X - Y` from `foo` — just like `Import foo(Z).`
